frontend/config/i18n.js

```diff
@@ -0,0 +1,12 @@
+const NextI18Next = require('next-i18next').default
+
+module.exports = new NextI18Next({
+  defaultLanguage: 'en',
+  // Other languages will be added here once there are translations for them, e.g. ['es', 'de']
+  otherLanguages: ['de'],
+  // The "dev" fallback language will show the translation key, indicating there is a missing translation
+  fallbackLng: process.env.NODE_ENV === 'production' ? 'en' : 'dev',
+  detection: {
+    order: ['navigator', 'cookie'],
+  },
+})
```

Comment:
Is there any particular reason to not make this a `.ts` file?

Reply:
I tried making this and the server file both TypeScript files, but ran into some dead ends. Making the server file into ts required quite a few new libraries and it just didn't seem worth it for just one file. Then because the server file imports this file and is in the node environment/uses CommonJS, it requires the exports of this file to be different, and it had trouble when it was `.ts`

Reply:
Ok we can leave it like this for the time being 😊